app/controllers/tasks_controller.rb

```diff
@@ -61,54 +57,29 @@ def destroy
 
   private
 
+  def set_search
+    search = params[:search]
+    @created_before_days = search[:created_before_days] if search.is_a?(ActionController::Parameters)
+    @option = params[:option]
```

Comment:
Reading the code, I was unable to identify the meaning behind the "option". It only got clear when executing the code. What about something like "visibility"? Or "scope"? Both are not ideal but maybe better than just "option"?